pkgs/development/python-modules/salib/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
+}:
+let
+  finalAttrs = {
+    pname = "salib";
+    version = "1.5.1";
+    pyproject = true;
+
+    src = fetchPypi {
+      inherit (finalAttrs) pname version;
+      hash = "sha256-5KnDGbjdAplajcmD9XxFLLflttvUPnt4VskMtqMyu18=";
+    };
+
+    buildInputs = with python3Packages; [
+      hatchling
+      hatch-vcs
+    ];
+
+    build-system = with python3Packages; [ setuptools ];
+
+    dependencies = with python3Packages; [
+      numpy
+      scipy
+      matplotlib
+      pandas
+      multiprocess
+      pathos
+    ];
+
```

Comment:
Please enable tests with `pytestCheckHook` and `pythonImportsCheck`.

Reply:
I set `pythonImportsCheck` but i cannot set `pytestCheckHook` because it makes the build failed, it collects 0 tests (there are no tests in the pypi dist).

I should specify it with `doCheck = false` (`pythonImportsCheck` will still be executed).